onward/app/models/MostPopularNx2.scala

```diff
@@ -137,6 +154,7 @@ object OnwardItemNx2 {
       kickerText = content.header.kicker.flatMap(_.properties.kickerText),
       starRating = content.card.starRating,
       avatarUrl = None,
+      branding = content.branding(Edition.defaultEdition),
```

Comment:
```suggestion
      branding = content.branding(Edition(request)),
```
I think(?) we should take the edition of the request here, and conveniently it's already available!

Reply:
Done